cmd/zt_copy_s2smigration_test.go

```diff
@@ -51,23 +51,22 @@ const (
 var defaultS2SInvalideMetadataHandleOption = common.DefaultInvalidMetadataHandleOption
 
 func (s *cmdIntegrationSuite) SetUpSuite(c *chk.C) {
-	// skip cleaning up the S3 account when not necessary
-	if isS3Disabled() && gcpTestsDisabled() {
-		return
-	}
-
-	if s3Client, err := createS3ClientWithMinio(createS3ResOptions{}); err == nil {
-		cleanS3Account(c, s3Client)
-	} else {
-		// If S3 credentials aren't supplied, we're probably only trying to run Azure tests.
-		// As such, gracefully return here instead of cancelling every test because we couldn't clean up S3.
-		c.Log("S3 client could not be successfully initialised")
+	if !isS3Disabled() {
```

Comment:
What is the change here, condition wise it's still the same it feels. It's just earlier we had an early exit in case both S3 and GCP were disabled and now we do the reverse checks and exit. What is the motivation behind this change?